integration_tests/dbt_project.yml

```diff
@@ -1,24 +1,17 @@
 name: 'twitter_source_integration_tests'
-version: '0.10.0'
+version: '0.10.1'
 
 
 profile: 'integration_tests'
 config-version: 2
 snapshot-paths: ["snapshots"]
 
 vars:
-  # twitter_source:
   twitter_ads_schema: twitter_ads_source_integration_tests
 
-  # Commend OUT for docs generation
-  twitter_ads__line_item_report_passthrough_metrics:
-    - name: "conversion_purchases_metric"
-    - name: "conversion_custom_sale_amount"
-      alias: "custom_conversion_value"
-
```

Comment:
Were these removed just for the docs generator or another reason? If the former, we can actually keep them by doing:

```
 twitter_ads__line_item_report_passthrough_metrics: # @docs-ignore
    - name: "conversion_purchases_metric" # @docs-ignore
    - name: "conversion_custom_sale_amount" # @docs-ignore
      alias: "custom_conversion_value" # @docs-ignore
```

Reply:
Oh nice! Yes I removed them for the docs generator, so will add that in